Automattic/wp-calypso
# Update global font pairings

#### Changes proposed in this Pull Request

* Updates the font pairings available in the global styles editor plugin to match those in this [this spreadsheet](https://docs.google.com/spreadsheets/d/1NTwkQqkwj-ekNaO0eRD4Cmt3pFGUXswfh_tFDbt_Qw8/edit#gid=1018936784) -- this matches the pairings available in Gutenboarding
* Also updates the UI to match the experience in Gutenboarding:

![font-pairings](https://user-images.githubusercontent.com/63313398/97618527-5be27780-19dc-11eb-9ce4-034109520049.gif)


#### Testing instructions

Check out the branch and sync to your sandbox using `yarn dev --sync`
Create a new post and open the Global Styles editor.

- [ ] Verify that the five font pairings displayed match what is listed in the spreadsheet
- [ ] Verify that the UI has been updated to match Gutenboarding
- [ ] Select a new font pairing and verify that the text in the post updates
- [ ] Select 'Publish' in the global styles sidebar and verify that the new font selection persists when the page is refreshed
- [ ] Select a different heading or base font from the Font Selection panel and verify that the selection (blue outline) is removed from the Font Pairings window
- [ ] Manually select a font pairing by making the Heading and Base font selections from the Font Selection window (eg manually select "Cabin" and "Raleway" from the dropdowns) and verify that the matching font pairing is selected in the UI (eg, the blue outline appears around "Cabin / Raleway")

Fixes #73
